src/static/helpers/slas.ts

```diff
@@ -83,6 +90,8 @@ export const generateCodeChallenge = async (
  * @param parameters.redirectURI - the location the client will be returned to after successful login with 3rd party IDP. Must be registered in SLAS.
  * @param parameters.hint? - optional string to hint at a particular IDP. Guest sessions are created by setting this to 'guest'
  * @param parameters.usid? - optional saved SLAS user id to link the new session to a previous session
+ * @param options - an object containing the options for this function.
+ * @param options.headers - optional headers to pass in the ShopperLogin authorizeCustomer method.
```

Comment:
Adding `?` for both `options` and `options.headers?` for consistency in doc comments for here and other functions please